API/Dockerfile

```diff
@@ -1,52 +1,24 @@
 #
-# Dockerfile for the prototype
+# Dockerfile for the API
 #
-#FROM ubuntu:latest
-#LABEL author="Ziad Khalaf"
-FROM alpine:latest AS builder
+FROM golang:latest AS builder
 USER root
-RUN apk add --no-cache git make musl-dev go
-
-# Configure Go
-ENV GOROOT /usr/lib/go
-ENV GOPATH /go
-ENV PATH /go/bin:$PATH
-
-RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
 WORKDIR /home
 
 ADD . /home/
-COPY ./resources/test/ /home/
-COPY ./.env /home/
-# RUN cd p3 && go mod init p3
-
 
 # Install Dependencies
-RUN go get -u github.com/gorilla/mux
-RUN go get -u go.mongodb.org/mongo-driver
-RUN go get -u github.com/dgrijalva/jwt-go
-RUN go get -u github.com/joho/godotenv
-RUN go get -u golang.org/x/crypto/bcrypt
-RUN go get go.mongodb.org/mongo-driver/x/mongo/driver@latest
-RUN go get go.mongodb.org/mongo-driver/x/mongo/driver/auth@latest
-RUN go get go.mongodb.org/mongo-driver/x/mongo/driver/ocsp@latest
-RUN go get go.mongodb.org/mongo-driver/x/mongo/driver/topology@latest
-RUN apk add --no-cache python3 py3-pip
-RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/community py3-pip
+RUN apt install git && go install .
```

Comment:
Why do we need to add git?

Reply:
Git is required in the build process here to provide values for our versioning variables (BuildTime, GitCommitDate)